### PR TITLE
Add agent token details to EventBridge agent events

### PR DESCRIPTION
**Pull request by jayco**
We are now sending the agent token UUID, created_at and description through in the EventBridge agent events.

### CODE REVIEW

File: pages/integrations/amazon_eventbridge.md.erb [hunk 1]
```diff
@@ -438,6 +438,11 @@ exports.handler = (event, context, callback) => {
       "uuid": "a98961b7-adc1-41aa-8726-cfb2c46e42e0",
       "graphql_id": "T3JnYW5pemF0aW9uLS0tYTk4OTYxYjctYWRjMS00MWFhLTg3MjYtY2ZiMmM0NmU0MmUw",
       "slug": "my-org"
+    },
+    "token": {
+      "uuid": "df75860c-94f9-4275-91cb-3986590f45b5",
+      "created_at": "2019-08-10 07:44:40 UTC",
+      "description": "my org token"
```
Comment (toolmantim) [inline pages/integrations/amazon_eventbridge.md.erb]:
```suggestion
      "description": "Default agent token"
```

File: pages/integrations/amazon_eventbridge.md.erb [hunk 2]
```diff
@@ -471,6 +476,11 @@ exports.handler = (event, context, callback) => {
...
```
Comment (toolmantim) [inline pages/integrations/amazon_eventbridge.md.erb]:
```suggestion
      "description": "Default agent token"
```

File: pages/integrations/amazon_eventbridge.md.erb [hunk 3]
```diff
@@ -504,6 +514,11 @@ exports.handler = (event, context, callback) => {
...
```
Comment (toolmantim) [inline pages/integrations/amazon_eventbridge.md.erb]:
```suggestion
      "description": "Default agent token"
```

File: pages/integrations/amazon_eventbridge.md.erb [hunk 4]
```diff
@@ -537,6 +552,11 @@ exports.handler = (event, context, callback) => {
...
```
Comment (toolmantim) [inline pages/integrations/amazon_eventbridge.md.erb]:
```suggestion
      "description": "Default agent token"
```

File: pages/integrations/amazon_eventbridge.md.erb [hunk 5]
```diff
@@ -570,6 +590,11 @@ exports.handler = (event, context, callback) => {
...
```
Comment (toolmantim) [inline pages/integrations/amazon_eventbridge.md.erb]:
```suggestion
      "description": "Default agent token"
```

**Comment by harrietgrace** [comment]
Thanks @jayco and @toolmantim! I've added in your changes @toolmantim 👍🏻

**Pull request merged**